### PR TITLE
Fix ignores in PointModelTest and AnswerModelTest

### PR DESCRIPTION
**Pull request by AnyaMaz**
#288 
- [x] -fix ignore of "Answer"
- [x] -fix ignore of "Point"

### CODE REVIEW

**Comment by malinink** [comment]
@AnyaMaz merge dev plz

**Comment by malinink** [comment]
@AnyaMaz merge dev!!!!

**Comment by AnyaMaz** [comment]
@malinink check pls...

File: src/test/java/easytests/support/meanbean/models/QuestionModelFactory.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import org.meanbean.lang.Factory;
+
+
+/**
+ * @author AnyaMaz
+ */
+public class QuestionModelFactory implements Factory<QuestionModelInterface> {
+
+    @Override
+    public QuestionModelInterface create() {return new QuestionModel();}
```
Comment (malinink) [inline src/test/java/easytests/support/meanbean/models/QuestionModelFactory.java]: cs make that in 3 lines on code.

**Review by malinink — CHANGES_REQUESTED** [review]

**Review by malinink — APPROVED** [review]

**Pull request merged**